js/pihole/index.js

```diff
@@ -182,36 +182,32 @@ function objectToArray(p){
 // Functions to update data in page
 
 function updateSummaryData(runOnce) {
-    $.getJSON("api.php?summary", function LoadSummaryData(data) {
-        //$("h3.statistic").addClass("glow");
-        if ($("h3#ads_blocked_today").text() != data.ads_blocked_today) {
-            $("h3#ads_blocked_today").addClass("glow");
-        }
-        if ($("h3#dns_queries_today").text() != data.dns_queries_today) {
-            $("h3#dns_queries_today").addClass("glow");
-        }
-        if ($("h3#ads_percentage_today").text() != data.ads_percentage_today) {
-            $("h3#ads_percentage_today").addClass("glow");
+    var setTimer = function(timeInMs) {
+        if (!runOnce) {
+            setTimeout(updateSummaryData, timeInMs);
         }
+    };
+    $.getJSON("api.php?summary", function LoadSummaryData(data) {
 
-        window.setTimeout(function(){
+        ["ads_blocked_today", "dns_queries_today", "ads_percentage_today"].forEach(function(today) {
+            var todayElement = $("h3#" + today);
+            todayElement.text() !== data[today] && todayElement.addClass("glow");
+        });
+
+        window.setTimeout(function() {
             $("h3#ads_blocked_today").text(data.ads_blocked_today);
             $("h3#dns_queries_today").text(data.dns_queries_today);
             $("h3#domains_being_blocked").text(data.domains_being_blocked);
             $("h3#ads_percentage_today").text(data.ads_percentage_today + "%");
```

Comment:
You should rewrite this in the same way you did in line [192](https://github.com/pi-hole/AdminLTE/pull/221/files#diff-f34e57b7cd3cc0747798f8caa65bfe22R192)